app/src/main/java/org/oppia/android/app/player/state/StateFragment.kt

```diff
@@ -79,12 +82,17 @@ class StateFragment :
     val storyId = arguments!!.getStringFromBundle(STATE_FRAGMENT_STORY_ID_ARGUMENT_KEY)!!
     val explorationId =
       arguments!!.getStringFromBundle(STATE_FRAGMENT_EXPLORATION_ID_ARGUMENT_KEY)!!
+    var rawUserAnswer: RawUserAnswer? = null
+    if (savedInstanceState != null) {
+      rawUserAnswer = savedInstanceState.getProto("Answer", RawUserAnswer.getDefaultInstance())
+    }
```

Comment:
```suggestion
    val rawUserAnswer = savedInstanceState?.getProto("Answer", RawUserAnswer.getDefaultInstance())
```

Can made safer, more concise, and more idiomatic.

Reply:
Done